pkgs/misc/themes/flat-plat/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchurl, gnome_themes_standard }:
+
+stdenv.mkDerivation rec {
+  name = "flat-plat-gtk-theme-${version}";
+  version = "3.20.20160404";
+
+  src = fetchurl {
```

Comment:
If this is a tarball uploaded by the project, I *think* github will leave it alone and serve it as-is, so the hash *should* be stable (compared to their auto-generated tarballs).